metron-analytics/metron-profiler-spark/README.md

```diff
@@ -60,6 +61,11 @@ For an introduction to the Profiler, see the [Profiler README](../metron-profile
       "timestampField": "timestamp"
     }
     ```
+    or
+
+    ii. Choose to use profiles already loaded into zookeeper (e.g. for use by the storm profiler bolt) by setting the environment variable `$SPARK_PROFILER_USE_ZOOKEEPER`. This will cause the spark profiler to extract profiles from the zookeeper quorum located at `$ZOOKEEPER`. 
```

Comment:
This is good documentation that we need, but it is a bit verbose for a `Getting Started' section.  

What if you moved these details into a separate section focused on how to load profile definitions; like 'Defining Profiles'.  Then in this section, just briefly mentioned that profile definitions can be loaded from either the filesystem or Zk and then link to the 'Defining Profiles' section.

Reply:
Docs restructured.